spec/board_spec.rb

```diff
@@ -8,13 +8,24 @@
       board = Board.new
       expect(board).to be_an_instance_of(Board)
     end
+
+    it "prints a welcome message" do
+      board = Board.new
+      def print_welcome
+        puts "Welcome to connect 4!"
+        puts "Select a column (ABCDEFG)"
+        puts " "
+      end
+      expect(board.print_welcome).to eq(print_welcome)
+    end
+    
     it "contains an empty board board" do
       board = Board.new
       expect(board.board[:row0]).to eq(['A', 'B', 'C', 'D', 'E', 'F', 'G'])
```

Comment:
This test could be made more robust by adding an enumerator to collect the symbols of each row into an array, and "expect" each array to "eq" [' . ', ' . ', ' . ', ' . ', ' . ', ' . ']